templates/test/ci/cluster-template-prow-ci-version.yaml

```diff
@@ -150,7 +150,7 @@ spec:
             $${GSUTIL} cp "$$CI_URL/$$CI_CONTAINER.$$CONTAINER_EXT" "$$CI_DIR/$$CI_CONTAINER.$$CONTAINER_EXT"
             $${SUDO} ctr -n k8s.io images import "$$CI_DIR/$$CI_CONTAINER.$$CONTAINER_EXT" || echo "* ignoring expected 'ctr images import' result"
             $${SUDO} ctr -n k8s.io images tag k8s.gcr.io/$$CI_CONTAINER-amd64:"$${CI_VERSION//+/_}" k8s.gcr.io/$$CI_CONTAINER:"$${CI_VERSION//+/_}"
```

Comment:
As far as I can see you can also drop those lines as `k8s.gcr.io` is used. But I'm not 100% sure, maybe worth a try to get rid of those references.

Reply:
Okay found it: https://github.com/kubernetes/kubernetes/commit/8ba5a06139d3d27f0d83c2d6afae0daeb3734d0e

So looks like kubeadm >=20 (?) uses the new repo.

So if we want this to work with kubeadm <= 1.19 from ci we would have to tag both? (not sure if we want that though)

Reply:
It was cherry-picked pretty far back. So the newest version which has the old ci repo is v1.17.*. So I think that's not relevant as we're (probably) not testing 1.17 CI versions anywhere anymore.